android/app/src/main/java/com/woowacourse/ody/presentation/meetinglist/BindingAdapter.kt

```diff
@@ -0,0 +1,61 @@
+package com.woowacourse.ody.presentation.meetinglist
+
+import android.view.View
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+import com.woowacourse.ody.R
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.Locale
+
+@BindingAdapter("setVisible")
+fun View.setVisible(isVisible: Boolean) {
+    visibility =
+        if (isVisible) {
+            TextView.VISIBLE
+        } else {
+            TextView.GONE
+        }
+}
+
+@BindingAdapter("showDateTime")
+fun TextView.showDateTime(dateTime: LocalDateTime) {
+    val meetingDay = dateTime.toLocalDate()
+    val today = LocalDateTime.now().toLocalDate()
+    val tomorrow = today.plusDays(1)
+    val isToday = meetingDay == today
+    val isTomorrow = meetingDay == tomorrow
+
+    val dateString =
+        when {
+            isToday -> {
+                this.context.getString(R.string.intro_today)
+            }
+
+            isTomorrow -> {
+                this.context.getString(R.string.intro_tomorrow)
+            }
+
+            (
+                meetingDay >= today.plusDays(2) &&
+                    meetingDay <= today.plusDays(7)
+            ) -> {
+                this.context.getString(
+                    R.string.intro_post_tomorrow,
+                    meetingDay.dayOfYear - today.dayOfYear,
+                )
+            }
+
+            else -> {
+                DateTimeFormatter.ofPattern("yyyy년 M월 d일").withLocale(Locale.forLanguageTag("ko")).format(dateTime)
+            }
+        }
+    val meetingTime = dateTime.toLocalTime()
+    val timeString =
+        if (isToday || isTomorrow) {
+            DateTimeFormatter.ofPattern("a h시 mm분").withLocale(Locale.forLanguageTag("ko")).format(meetingTime)
+        } else {
+            ""
```

Comment:
여기도 동일합니다. `withLocale(Locale.forLanguageTag("ko"))`!!

Reply:
```suggestion
    if (!isToday && !isTomorrow) {
        this.text = dateString
        return
    }
    val timeString = DateTimeFormatter.ofPattern("a h시 mm분").withLocale(Locale.forLanguageTag("ko")).format(meetingTime)
    this.text = "$dateString $timeString"
```
이런 방식은 어떤가요? early return 매우 선호하는 편....

Reply:
early return 으로 바꿔주었습니다.
Locale을 없애주면 아마 AM, PM으로 출력이 될거에요.